addons/payment/tests/utils.py

```diff
@@ -12,13 +12,6 @@
 
 
 class PaymentTestUtils(AccountTestInvoicingCommon):
-
-    @classmethod
-    def setUpClass(cls, chart_template_ref=None):
-        super().setUpClass(chart_template_ref=chart_template_ref)
```

Comment:
I expect it was introduced for the `base_url` attribute. Since that was removed, it doesn't do anything anymore.

*If* it needs to be reintroduced in the future, it can be.